lib/internal/bootstrap/pre_execution.js

```diff
@@ -119,7 +119,11 @@ function setupCoverageHooks(dir) {
   const cwd = require('internal/process/execution').tryGetCwd();
   const { resolve } = require('path');
   const coverageDirectory = resolve(cwd, dir);
-  internalBinding('profiler').setCoverageDirectory(coverageDirectory);
+  try {
+    internalBinding('profiler').setCoverageDirectory(coverageDirectory);
+  } catch {
+    process.emitWarning('Profiler is not enabled.', 'Warning');
+  }
```

Comment:
Also should the return value here be `null`?

Reply:
@shobhitchittora It’s used as the new value of `process.env.NODE_V8_COVERAGE` when returned, so using an empty string (which would disable coverage for subprocesses automatically) would be okay, I’d say.

Reply:
Sure @addaleax! makes sense. Returned an empty string.